src/main/java/com/dreammaster/gthandler/GT_Recipe_Remover.java

```diff
@@ -4509,6 +4520,17 @@ public void run() {
                 getModItem("witchery", "ingredient", 1, 130, missing),
                 getModItem("minecraft", "magma_cream", 1, 0, missing),
                 getModItem("minecraft", "blaze_powder", 1, 0, missing));
+        removeRecipeShapelessDelayed(
+                "ingotThauminite",
+                getModItem("thaumicbases", "resource", 1, 0, missing),
+                getModItem("thaumicbases", "resource", 1, 0, missing),
+                getModItem("thaumicbases", "resource", 1, 0, missing),
+                getModItem("thaumicbases", "resource", 1, 0, missing),
+                getModItem("thaumicbases", "resource", 1, 0, missing),
+                getModItem("thaumicbases", "resource", 1, 0, missing),
+                getModItem("thaumicbases", "resource", 1, 0, missing),
+                getModItem("thaumicbases", "resource", 1, 0, missing),
+                getModItem("thaumicbases", "resource", 1, 0, missing));
```

Comment:
I know it's not done at all in general here but I wonder why it's done like this. 🤔 
I personally would save the item to a variable and re-use instead of calling the GameRegistry.findItem() all the time. Not sure how that affects the cpu usage while registring the recipes btw.

Reply:
Ah ok.